### PR TITLE
t2: add missing help to crash-reporter. Fixes gh-708

### PR DESCRIPTION
**Pull request by rwaldron**
```
$ t2

command argument is required

Usage: t2 <command>

command
  install-drivers     Install drivers
  crash-reporter      Crash Reporter enable/disable
  provision           Authorize your computer to control the USB-connected Tessel
  restart             Restart a previously deployed script in RAM or Flash memory (does not rebundle)
  run
    Deploy a script to Tessel and run it with Node.

    Assets that are not directly deployed as a dependency via require analysis,
    for example images or html files of an application (and their directories),
    must be listed in a .tesselinclude in the root of your project.
    This can be created manually or by typing 't2 init'.

    For more information, visit: https://tessel.io/docs/cli#starting-projects

  push
    Pushes the file/dir to Flash memory to be run anytime the Tessel is powered,
    runs the file immediately once the file is copied over.

    Assets that are not directly deployed as a dependency via require analysis,
    for example images or html files of an application (and their directories),
    must be listed in a .tesselinclude in the root of your project.
    This can be created manually or by typing 't2 init'.

    For more information, visit: https://tessel.io/docs/cli#starting-projects

  erase               Erases files pushed to Flash using the tessel push command
  list                Lists all connected Tessels and their authorization status.
  init                Initialize repository for your Tessel project
  wifi                Configure the wireless connection
  key                 Manage ssh keys for connecting to a Tessel
  rename              Change the name of a Tessel to something new
  update              Update the Tessel firmware and openWRT image
  version             Display Tessel's current firmware version
  ap                  Configure the Tessel as an access point
  root                Gain SSH root access to one of your authorized tessels
```

cc @Frijol 

Signed-off-by: Rick Waldron <waldron.rick@gmail.com>

### CODE REVIEW

**Comment by wyze** [comment]
LGTM

**Comment by rwaldron** [comment]
@wyze thanks!

File: bin/tessel-2.js [hunk 1]
```diff
@@ -115,7 +115,8 @@ parser.command('crash-reporter')
   .option('test', {
     flag: true,
     help: 'Test the Crash Reporter.'
-  });
+  })
+  .help('Crash Reporter enable/disable');
```
Comment (Frijol) [inline bin/tessel-2.js]: How do you know if it's enabled or disabled? Do you run it with --enable or anything like that?

Comment (wyze) [inline bin/tessel-2.js]: A few lines above it there is `--on` and `--off` flags. Not sure how to get the current state, or if there is a way or not.

Reference: https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L107-L114

**Comment by tikurahul** [comment]
Its enabled by default. There is a way to toggle it from the command line. There is nothing that will tell you it's current state via the command line, but we can figure that out programmatically very easily via `preferences.js`. It should be a really easy feature to add, if we need it.

**Pull request merged**